include/z64.h

```diff
@@ -785,7 +785,7 @@ typedef struct {
     /* 0x022C */ s16    cursorY[5]; // "cur_ypt"
     /* 0x0236 */ s16    dungeonMapSlot;
     /* 0x0238 */ s16    cursorSpecialPos; // "key_angle"
-    /* 0x023A */ s16    pageSwitchTimer;
+    /* 0x023A */ s16    delaySwitchPageInputTimer; // Used to introduce a delay before switching page when arriving on the "scroll left/right" positions while holding stick left/right.
```

Comment:
kinda feel like we dont need all of these words to convey this meaning. `pageDelayTimer` for example seems fine to me, or even the current name

Reply:
The current name doesn't work because there is another timer involved in switching between pages so that would be two "page switch timer"s

I couldn't think of a satisfying shorter name but I'm open to suggestions

Reply:
For pageDelayTimer I feel like that would miss the two words,
- Switch because without it the context/usage is unclear, it could be a timer for anything
- Input because this is a really important feature of processing the inputs, if this timer wasn't implemented you'd need to do single-frame x inputs to move the cursor on the left/right special pos without actually scrolling to the next page

Reply:
I would have gone with switchPageInputDelay if "input delay" didn't refer to "input lag" which gives the wrong idea

Reply:
we could start using "scroll" to refer to "switch page" instead, that would result in `delayScrollInputTimer`

Also maybe it would be fine to drop the "delay" in front though I'd rather not

Reply:
Late to the discussion but I think I agree with Fig here.

I guess we can't use the current name if there is another timer involved, but we probably don't need both "input" and "delay" to differentiate it. I suggest going with either `pageSwitchDelayTimer` or `pageSwitchInputTimer`, since those feel the most natural to me while still being clear enough